backend/src/services/__tests__/game.service.test.ts

```diff
@@ -82,14 +88,40 @@ describe("validateGameCode Service", () => {
   it("returns false for an invalid gameCode", async () => {
     const result = await validateGameCode("69420");
 
-    expect(result).toBe(false);
+    expect(result).toBeFalsy();
```

Comment:
I feel like all these service tests should be part of a separate PR as they muddy the review and make it hard to go through all the files.

The changes I see in the services seem to be mainly refactors so I'm not sure that they need to be included in the PR. The additional tests thus seem geared toward code coverage, which I think means they should be part of a separate change.

Let me know if I've missed something however

Reply:
Yeah I agree it would've been nicer to split these up. Unfortunately I had to make changes to the services to implement the handlers and I made some additionally refactors to the services to ensure consistency